docs/en/sql-reference/statements/insert-into.md

```diff
@@ -13,12 +13,60 @@ Basic query format:
 INSERT INTO [db.]table [(c1, c2, c3)] VALUES (v11, v12, v13), (v21, v22, v23), ...
 ```
 
-The query can specify a list of columns to insert `[(c1, c2, c3)]`. In this case, the rest of the columns are filled with:
+You can specify a list of columns to insert using  the `(c1, c2, c3)` or `COLUMNS(c1,c2,c3)` syntax. 
+You can use asterisk when specifying the column list.
```

Comment:
```suggestion
You can specify a list of columns to insert using  the `(c1, c2, c3)` or `COLUMNS(c1,c2,c3)` syntax. 

Instead of listing all the required columns you can use the `(* EXCEPT(column_list))` syntax.
```